lib/Extension/LanguageServerReferenceFinder/Model/Highlighter.php

```diff
@@ -132,6 +134,17 @@ private function variableKind(Node $node): int
     private function properties(Node $rootNode, string $name): Generator
     {
         foreach ($rootNode->getDescendantNodes() as $node) {
+            if ($node instanceof Parameter && null !== $node->visibilityToken && (string)$node->getName() === $name) {
+                yield new DocumentHighlight(
+                    new Range(
+                        PositionConverter::intByteOffsetToPosition($node->variableName->getStartPosition(), $node->getFileContents()),
+                        PositionConverter::intByteOffsetToPosition($node->variableName->getEndPosition(), $node->getFileContents())
+                    ),
+                    DocumentHighlightKind::TEXT
+                );
+                continue;
```

Comment:
Maybe it would be good to add `continue` also below the second `yield` to skip further type checking - WDYT @dantleech?